dealers_choice.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <title>Top 5 Sushi Restaurants in NYC</title>
+        <link href="dealers_styles.css" rel="stylesheet" />
+        <script src="dealers.js" defer></script>
+    </head>
+    <body>
+
+        <header>
+        <ul class="nav"> 
+             <li>
+                <a target = "_blank" href="https://delishably.com/meat-dishes/The-Different-Kinds-of-Sushi">Types of Sushi</a> 
+             </li>
+             <li>
+                <a target = "_blank" href="https://guide.michelin.com/us/en/article/features/kitchen-language-what-is-omakase">Omakase</a> 
+             </li>
+             <li>
+                <a target = "_blank" href="https://www.rokaakor.com/new-to-sushi-a-simple-guide-to-eating-sushi-for-beginners/">Guide to Sushi</a> 
+             </li>
+             <li>
+                <a target= "_blank" href="https://rbsushi.com/blog/8-most-popular-types-of-fish-served-as-sashimi/">Common Fish</a>
+             </li>
+             <li>
+                <a target= "_blank" href="https://www.tripadvisor.com/Restaurants-g60763-c38-New_York_City_New_York.html">Reviews</a>
+             </li>
+        </ul>
+        </header>
+
+        <section id="carousel">
+            <div id="carousel-text">
+              <h1>Top 5 Sushi Restaurants in NYC </h1>
+            </div>
+            <img class="carousel-image" src="https://popmenucloud.com/cdn-cgi/image/width%3D1200%2Cheight%3D1200%2Cfit%3Dscale-down%2Cformat%3Dauto%2Cquality%3D60/bqynrcpd/4c999b07-82e9-46c0-b675-4b6e2236b146.jpg" />
+            <img class="carousel-image hidden" src="https://pyxis.nymag.com/v1/imgs/d06/96c/53a34497349795cbe22d2a00a6526cfd08-bony-sushi-noz.rhorizontal.w700.jpg" />
+            <img class="carousel-image hidden" src="https://hijinkslife.com/blog/wp-content/uploads/2017/03/New-York-City-Sushi-Class.jpg" />
+            <img class="carousel-image hidden" src="https://robbreport.com/wp-content/uploads/2020/08/juku-sushi-takeout.jpg?w=1000" />
+          </section>
+
+        <div class="first_post">
+            <p>
+                When you're looking for the top sushi in NYC, here's where to go. These are the omakase spots where you'll feel like you're at a day spa, the neighborhood staples that go above and beyond, and the restaurants where you can and should be celebrating birthdays. Are some of these places wildly expensive? Absolutely. But there are also a few options on this list where you can get out for under $50. No matter what you're paying, you can be confident that it's worth it. 
+            </p>
+            </div>
```

Comment:
There is a closing div in line 46. But I am not sure how this is being used. I don't see the opening div.